Sources/Configuration/ConfigurationFetching.swift

```diff
@@ -74,7 +74,7 @@ public final class ConfigurationFetcher: ConfigurationFetching {
       An error of type Error is thrown if the configuration fails to fetch or validate.
     */
     public func fetch(_ configuration: Configuration) async throws {
-        let fetchResult = try await fetch(from: configuration.url, withEtag: etag(for: configuration), requirements: .default)
+        let fetchResult = try await fetch(from: configuration.url, withEtag: etag(for: configuration), requirements: .all)
```

Comment:
This was missing the `.allowHTTPNotModified` requirement, which is an expected situation for the config fetch API.

Let me know if you'd rather that I be explicit with the requirements here, as using `.all` may lead to a situation where unexpected requirements are placed upon this call when new ones are added in the future.

Reply:
That makes sense, thanks @jaceklyp – I'll push this change down to the client side and check for the specific error type as you suggested. As a result this PR is no longer needed so I'll close it.